jkube-kit/doc/src/main/asciidoc/inc/build/_overview.adoc

```diff
@@ -113,3 +145,100 @@ The following example replaces all properties in the format `@property@` within
     }
 }
 ----
+endif::[]
+ifeval::["{plugin-type}" == "maven"]
+[source,xml,indent=0,subs="verbatim,quotes,attributes"]
+----
+<plugin>
+ <configuration>
+   <images>
+     <image>
+       <name>user/demo</name>
+       <build>
+         <filter>@</filter>
+       </build>
+     </image>
+   </images>
+ </configuration>
+ ...
+</plugin>
+----
+endif::[]
+
+[[jkube-plugin]]
```

Comment:
JKube plugins deserve a complete separate section, maybe similar to profiles or any other generic feature JKube offers.

We can then link each operation section to its relevant JKubePlugin parts